Index: admin/company_preferences.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/company_preferences.php	(date 1499497680000)
+++ admin/company_preferences.php	(revision )
@@ -173,6 +173,7 @@
 
 text_row_ex(_("Official Company Number:"), 'coy_no', 25);
 text_row_ex(_("GSTNo:"), 'gst_no', 25);
+text_row_ex(_("VAT No:"), 'vat_no', 25);
 
 currencies_list_row(_("Home Currency:"), 'curr_default', $_POST['curr_default']);
 fiscalyears_list_row(_("Fiscal Year:"), 'f_year', $_POST['f_year']);
